dataprofiler/profilers/profile_builder.py

```diff
@@ -916,7 +1061,7 @@ def load(cls, filepath):
         return profile
 
 
-class Profiler(object):
+class Profiler(BaseProfiler):
```

Comment:
should it be `StructuredProfiler` now?

Reply:
That is not the intent of this task, hence, no. @ChrisWallace2020 is working on that task.